Week2/6.js

```diff
@@ -0,0 +1,11 @@
+myArray = [];
```

Comment:
are you sure that this will work? (without `const` or `let`)?

Reply:
it is completely overlooked issue in a busy time. so it should be `let myArray = [];` or `const myArray = [];`.

Reply:
I think `const` is fine